database/index.js

```diff
@@ -1,12 +1,12 @@
 const { Client } = require('pg');
-const { USERNAME, PASSWORD } = require('../config.js');
+const { USERNAME, DATABASE, PASSWORD, HOST, PORT } = require('../config.js');
 
 const client = new Client({
   user: USERNAME,
-  database: 'best_eats',
+  database: DATABASE,
   password: PASSWORD,
-  host: 'localhost',
-  port: 5432,
+  host: HOST,
+  port: PORT,
 });
```

Comment:
We could probably just import the object with all the data rather than all the individual variables.